### PR TITLE
Add main service for boxi export job

### PR DESCRIPTION
**Pull request by cintamani**
From: https://eaflood.atlassian.net/browse/RUBY-738

This includes:
 - New zip gem to create zip files
 - Implementation of the main service
 - Add rake task
- Add Boxi bucket configurations

Todo:
 - Implement serializers

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -33,6 +33,7 @@ WCRS_EMAIL_SERVICE_EMAIL=""
 AWS_DAILY_EXPORT_ACCESS_KEY_ID=
 AWS_DAILY_EXPORT_SECRET_ACCESS_KEY=
 AWS_DAILY_EXPORT_BUCKET=
+AWS_BOXI_EXPORT_BUCKET=
```
Comment (Cruikshanks) [inline .env.example]: Only comment is that these need to come in 3's

- Access Key ID
- Secret Access Key
- Boxi export bucket

We save a bit of time and money by using the same credentials and buckets in non-production environments, but often they are different when it comes to production.

So we need the flexibility to specify a different access key ID and secret access key for BOXI, as well as the bucket name.

Comment (cintamani) [reply .env.example]: WIll add them in this PR :)

**Comment by cintamani** [comment]
I am not sure what's actually wrong with Hakiri 🤔

**Comment by irisfaraway** [comment]
@cintamani The Rack gem has a security vulnerability. I'll bump Dependabot and see if that updates it. Been patching other projects this morning.

**Comment by irisfaraway** [comment]
Although actually looks like it should be fixed here: https://github.com/DEFRA/waste-carriers-back-office/pull/523

A rebase might fix it?

**Comment by cintamani** [comment]
Just rebased! 🤞

**Pull request merged**